edb\_exception
==============

.. currentmodule:: pyedb.dotnet.edb

.. automethod:: Edb.edb_exception